charts/argo-cd/templates/argocd-configs/argocd-secret.yaml

```diff
@@ -34,6 +34,10 @@ data:
   {{- with .Values.configs.secret.gogsSecret }}
   webhook.gogs.secret: {{ . | b64enc }}
   {{- end }}
+  {{- with .Values.configs.secret.azureDevops }}
+  webhook.azuredevops.username: {{ .username }}
+  webhook.azuredevops.password: {{ .password | b64enc }}
+  {{- end }}
```

Comment:
📝 
Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook/#2-configure-argo-cd-with-the-webhook-secret-optional

<img width="487" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/fe2c4d38-140c-436c-abd8-ea3b773ade4f">